ci-templates/.github/workflows/deploy-to-pantheon.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy to Pantheon
+
+on:
+  push:
+    branches:
+      - production
+      - preprod
+      - develop
+
+jobs:
+  build-and-sync:
+    runs-on: ubuntu-latest
+    name: "Build and Deploy to Pantheon"
+    timeout-minutes: 10
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        submodules: 'recursive'
```

Comment:
We shouldn't be using submodules at all - can we nix this line?

Reply:
np. just not fully removed from previous example. I'll nuke!